30413/Andrei Ionut Lauran/Lab02/README.md

```diff
@@ -3,4 +3,5 @@
 > Edit this file and describe in a few words what you've done in this lab.
 
 I have tried to solve the mandatory problem without succes. I watched some tutorials and read the pdf file few times tho
-and managed to understand the concept of single linked list but i still couldn't solve the problem. I will try again.
+and managed to understand the concept of single linked list but i still couldn't solve the problem. I will try again, maybe
+i'll have some new ideeas
```

Comment:
You gotta keep trying. This is how you learn to write code and debug it. Never give up on an issue.